src/Confluent.Kafka/Impl/SafeKafkaHandle.cs

```diff
@@ -565,6 +565,50 @@ internal Task<CommittedOffsets> CommitAsync()
         internal Task<CommittedOffsets> CommitAsync(IEnumerable<TopicPartitionOffset> offsets)
             => Task.Run(() => commitSync(offsets));
 
+        internal List<TopicPartitionError> Pause(IEnumerable<TopicPartition> partitions)
+        {
+            ThrowIfHandleClosed();
+            IntPtr list = LibRdKafka.topic_partition_list_new((IntPtr) partitions.Count());
```

Comment:
The IEnumerable<TopicPartition*> to partition_list_t mangling (and reverse) is pretty common, might want to refactor into a helper method to avoid code duplication.